packages/react-server/src/ReactFlightServer.js

```diff
@@ -636,7 +664,6 @@ export function resolveModelToJSON(
 
         return serializeByRefID(newTask.id);
       } else {
-        logRecoverableError(request, x);
```

Comment:
@gnoff This seems like it was a duplicate call. Not sure if in intentional.

Reply:
👀  wow, not sure how I let that one stick around. def not intional